catalogue_pipeline/id_minter/src/test/scala/uk/ac/wellcome/platform/idminter/modules/IdMinterWorkerTest.scala

```diff
@@ -2,64 +2,68 @@ package uk.ac.wellcome.platform.idminter.modules
 
 import java.sql.SQLSyntaxErrorException
 
-import org.scalatest.FunSpec
+import com.twitter.finatra.http.EmbeddedHttpServer
+import org.scalatest.{FunSpec, Matchers}
+import org.scalatest.concurrent.Eventually
 import org.scalatest.mockito.MockitoSugar
 import scalikejdbc._
-import scalikejdbc.interpolation.SQLSyntax
 import uk.ac.wellcome.platform.idminter.database.{
   FieldDescription,
   IdentifiersDao
 }
-import uk.ac.wellcome.platform.idminter.utils.IdMinterTestUtils
+import uk.ac.wellcome.platform.idminter.{fixtures, Server}
+import uk.ac.wellcome.test.fixtures.{SnsFixtures, SqsFixtures}
+import uk.ac.wellcome.test.utils.ExtendedPatience
 
 class IdMinterWorkerTest
     extends FunSpec
-    with IdMinterTestUtils
+    with SqsFixtures
+    with SnsFixtures
+    with fixtures.IdentifiersDatabase
+    with fixtures.Server
+    with Eventually
+    with ExtendedPatience
+    with Matchers
     with MockitoSugar {
 
-  private val identifiersDao = mock[IdentifiersDao]
-  private val server = defineServer.bind[IdentifiersDao](identifiersDao)
+  it("should create the Identifiers table in MySQL upon startup") {
+    withLocalSqsQueue { queueUrl =>
+      withLocalSnsTopic { topicArn =>
```

Comment:
Do we ever use an SNS topic without an SQS queue in our tests?

Reply:
Yes, there seem to be a few places, however most of the `...Feature` tests will always use both so there may be utility in combining these into a single trait (though i'd prefer to do that in a future PR).